packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -123,6 +123,29 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     this._tooltipId = getId('AreaChartTooltipID');
   }
 
+  public componentDidUpdate() {
+    if (this.state.isShowCalloutPending) {
+      const points = this.props.data.lineChartData!;
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      this._stackedData.forEach((singleStackedData: Array<any>, index: number) => {
+        if (points.length === index) {
+          return;
+        }
+        singleStackedData.forEach((singlePoint: IDPointType, pointIndex: number) => {
```

Comment:
Is there a better way than looping through all points here

Reply:
Other charts dont use isShowCalloutPending state at all. Let me see if there is a better way.

Reply:
I went through the code again but I am not able to find any better solution because of `circleId`